com.jetbrains.RustRover.appdata.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.jetbrains.RustRover</id>
+  <metadata_license>CC-BY-SA</metadata_license>
+  <project_license>LicenseRef-proprietary</project_license>
+  <name>RustRover</name>
+  <summary>A cross-platform IDE for Rust</summary>
+  <description>
+    <p>
+      RustRover is a cross-platform IDE that provides consistent experience on the Windows, macOS, and Linux operating systems.
```

Comment:
can you add some more description?

Reply:
hmmm there isn't much more description available at Jetbrains